DEVEL.rst

```diff
@@ -143,8 +187,8 @@ vi ~/.commissaire.json
 
 To query the known clusters:
 
-```
-commctl cluster list
-```
+.. code-block::
+
+   $ commctl cluster list
```

Comment:
This worked much better once the bus connection was up and running

```
$ commctl cluster list
DEBUG:StorageService:Received message "c502b9a5-2738-4dd9-a3ad-257608846822" {'id': '158fd848-fef4-4d02-82f2-9c69f836c6e8', 'jsonrpc': '2.0', 'method': 'list', 'params': {'model_type_name': 'Clusters'}}
DEBUG:store:> LIST <Clusters []...>
DEBUG:etcd.client:Issuing read for key /commissaire/clusters/ with args {'recursive': True}
DEBUG:urllib3.connectionpool:http://localhost:2379 "GET /v2/keys/commissaire/clusters/?recursive=true HTTP/1.1" 200 102
WARNING:EtcdStoreHandler:Etcd returned unserializable data. Skipping the record.TypeError: the JSON object must be str, not 'NoneType'
DEBUG:store:< LIST <Clusters []...>
DEBUG:StorageService:Result for "158fd848-fef4-4d02-82f2-9c69f836c6e8": "[]"
DEBUG:StorageService:Responding to response-158fd848-fef4-4d02-82f2-9c69f836c6e8
DEBUG:StorageService:Message "c502b9a5-2738-4dd9-a3ad-257608846822" was ackd
127.0.0.1 - - [04/Apr/2017 11:15:31] "GET /api/v0/clusters HTTP/1.1" 200 2
Success
...
```

Still seeing that common warning message though

```
WARNING:EtcdStoreHandler:Etcd returned unserializable data. Skipping the record.TypeError: the JSON object must be str, not 'NoneType'
```